### PR TITLE
chore: update tooltip text

### PR DESCRIPTION
**Pull request by tadejpodrekar**

### CODE REVIEW

File: apps/frontend/src/components/DashboardView/AssetsTable/BorrowTable.tsx [hunk 1]
```diff
@@ -57,17 +57,26 @@ const POINTS_BORROW: Point[] = [
     name: 'Activity Points',
     description: (
       <div className="text-md">
-        Earn up to <span className="text-primary">4x</span>{' '}
-        <span className="text-lavender font-semibold">Activity Points</span> by{' '}
-        borrowing <br /> assets on Swaylend.{' '}
+        By Borrowing USDC On Swaylend you get{' '}
```
Comment (martines3000) [inline apps/frontend/src/components/DashboardView/AssetsTable/BorrowTable.tsx]:
```suggestion
        By Borrowing USDC on Swaylend you get{' '}
```

File: apps/frontend/src/components/DashboardView/AssetsTable/LendTable.tsx [hunk 2]
```diff
@@ -54,17 +54,26 @@ const POINTS_LEND: Point[] = [
     name: 'Activity Points',
     description: (
       <div className="text-md">
-        Earn <span className="text-primary">4x</span>{' '}
-        <span className="text-lavender font-semibold">Activity Points</span> by
-        lending assets <br /> on Swaylend.{' '}
+        By Lending USDC On Swaylend you get{' '}
```
Comment (martines3000) [inline apps/frontend/src/components/DashboardView/AssetsTable/LendTable.tsx]:
```suggestion
        By Lending USDC on Swaylend you get{' '}
```

File: apps/frontend/src/components/DashboardView/AssetsTable/CollateralTable.tsx [hunk 3]
```diff
@@ -76,19 +76,41 @@ const POINTS_COLLATERAL: Point[] = [
...
     description: (
       <div className="text-md">
-        Earn <span className="text-lavender font-semibold">Passive Points</span>{' '}
-        by holding <span className="text-lavender">unused</span> collateral{' '}
-        <br /> on Swaylend. Note that Collateral used in <br /> Borrow positions
-        earns{' '}
-        <span className="text-lavender font-semibold">Activity Points</span>{' '}
-        instead. <br />{' '}
+        Supply this asset as collateral to earn 1 Fuel Point per dollar value.
+        <br /> Earn up to a <span className="text-primary">2x</span> multiplier
```
Comment (martines3000) [inline apps/frontend/src/components/DashboardView/AssetsTable/CollateralTable.tsx:80]:
```suggestion
        <br /> Earn a <span className="text-primary">2x</span> multiplier
```

Comment (tadejpodrekar) [reply apps/frontend/src/components/DashboardView/AssetsTable/CollateralTable.tsx:80]: This was the text that was provided

**Review by martines3000 — APPROVED** [review]

**Pull request merged**